_data/companies.yml

```diff
@@ -1533,12 +1533,12 @@
   events: Restricted
   last_update: 2020-03-13
 
-- name: Lightstep
+- name: Levi Strauss & Co.
```

Comment:
It looks like you're trying to overwrite Lightstep here, which I don't think you want. Would you mind fixing this?

Reply:
I think its resolved with #1342